css/vtm5e.css

```diff
@@ -540,6 +553,14 @@
 
 .roll-img {
   max-width: 10%;
+  margin: 1px;
+	background: rgba(255,255,255,0.5);
+	border-radius: 3px;
+}
+
+.roll-img.hunger-dice {
+  background: rgba(255,230,230,0.5);
+  border-color: #790813;
 }
```

Comment:
I was looking for a more distinct look of hunger dice for the chat :)

Reply:
![demo_hunger_dice](https://user-images.githubusercontent.com/1277339/123556456-a4308000-d78b-11eb-97a3-f4d15e01c389.jpg)